services/api-server/openapi.json

```diff
@@ -2220,7 +2220,7 @@
           }
         ],
         "responses": {
-          "200": {
+          "202": {
```

Comment:
This asks for a change in the API version. 
Since you are breaking backwards compatibility, you should increase major?

Reply:
Yep, makes sense. Done

Reply:
@mguidon I will bump the major version of the api-server to 1 instead of 0. This will probably require a change in your c++ client, but just in the base url you are pointing to. Let me know if you want me to make the change.

Reply:
OK, we have decided not to bump the major version, but only the minor version, so no need to change anything in the c++ client